bucket/goneovim.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "v0.4.12",
+    "description": "Neovim GUI written in Golang, using a Golang qt backend",
+    "url": "https://github.com/akiyosi/goneovim/releases/download/v0.4.12/Goneovim-v0.4.12-win64.zip",
+    "hash": "2FE23B57E9473E90D6FEA029B4AD3E4A89EABB8EC093AF4993115145F5625953",
+    "license": "MIT",
+    "homepage": "https://github.com/akiyosi/goneovim",
+    "bin": "goneovim.exe",
+    "shortcuts": [
+        [
+            "goneovim.exe",
+            "Goneovim"
+        ]
+    ],
+    "suggest": {
+        "Neovim": "neovim"
+    },
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/akiyosi/goneovim/releases/download/$version/Goneovim-$version-win64.zip"
```

Comment:
Could you add URL to architecture 64bit block?

Reply:
Check https://github.com/ScoopInstaller/Extras/blob/master/bucket/staxrip.json for example

Reply:
Check https://github.com/ScoopInstaller/Extras/blob/master/bucket/staxrip.json for example

Reply:
Further URL should have v$version I believe